### DISCUSSION TITLE
how to fix metal spacing DRCs

### DISCUSSION THREAD

**Discussion by gkamendje** [The-OpenROAD-Project/OpenROAD#3589] [Q&A] [answered]
After 64 iterations DRT stopped on my design a generated a DRC report. It turns out that most of the DRC are related to METAL1 spacing  (with net VSS) and Metal SpacingTableTw. The design is not very congested so I wonder why the tool could not fix these rules. Could it be that I am missing something here (maybe something in my platform definition)? Is there a way to fix these violations? 
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/56942214/af73acfe-92fb-4590-baaf-31be8548e95e)

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/56942214/4ed7f795-fef7-4d49-9f78-76b85c9cae37)

**Comment by maliberty** [comment]
Can you provide a test case?

**Reply by gkamendje** [reply]
@maliberty an attempt to upload a testcase failed because the file is too big (34 MB). Only 25 MB are allowed. Is there another way I could share it?

**Reply by maliberty** [reply]
You can upload to a service like Google drive and provide a link

**Reply by gkamendje** [reply]
@maliberty  Here is the link 
https://drive.google.com/file/d/1-jsIcU4Rs2zaGPJWNn_TkyjmjEpFfyGg/view?usp=sharing
Please let me know if you can access it.

**Reply by gkamendje** [reply]
@maliberty I was wondering if you had the chance to have a look at this and if I should keep the link active.

**Reply by maliberty** [reply]
Sorry this got overlooked.  I've downloaded it and will take a look.

**Comment by maliberty** [comment, marked as answer]
You have very wide METAL1 power stripes:

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/761514/ec146977-bcb3-4725-8140-4bef3c0137ef)

The pin is less than the min space (0.19 < 0.23) from the stripe so there is always going to be a DRC error here.   You need to adjust your PDN definition to be resolve this.
